mentat/llm_api.py

```diff
@@ -86,6 +86,17 @@ async def call_llm_api(
     return _add_newline(response)
 
 
+def call_embedding_api(
+    input: list[str], model: str = "text-embedding-ada-002"
+) -> list[list[float]]:
+    if is_test_environment():
+        logging.critical("OpenAI call attempted in non benchmark test environment!")
+        raise MentatError("OpenAI call attempted in non benchmark test environment!")
```

Comment:
instead of duplicating these lines from `call_llm_api` lets make a new function that both call, `raise_if_in_test_environment`